### DISCUSSION TITLE
Precision for Duration

### DISCUSSION THREAD

**Discussion by urhausen** [pytest-dev/pytest-html#515] [Q&A]
Hello, I was wondering about the precision of the duration column. Is there any reason why %f is restricted to a precision of 2?
We have many quick tests where the duration column shows values of 0.00, but I think it would be nice to increase the precision to see something like 0.003. Thank you!

**Comment by BeyondEvil** [comment]
Remind me when we've released v4.0.0.

I don't see why we wouldn't be able to display a higher precision.

Note however, that we're restricted by whatever precision `pytest` passes to us.

**Comment by BeyondEvil** [comment]
Please try 4.0.0rc1. Precision is now down to 1ms.
